common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/FederationBatch.java

```diff
@@ -29,17 +29,23 @@
   @Id
   private String batchTag;
 
-  private Date date;
+  private LocalDate date;
 
   private FederationBatchStatus status;
 
   /**
    * Creates a FederationBatch.
    */
-  public FederationBatch(String batchTag, Date date, FederationBatchStatus status) {
+  public FederationBatch() {
```

Comment:
Because this is a core domain class, it would probably be good to add a description of what this entity represents.

Reply:
Is the default constructor needed? Could we try to preserve imutability of this class and use private final fields for the entity properties combined with a constructor which initalizez all fields? If yes, we could also remove the setters.